### ISSUE TITLE
What exactly is "Plither"?

### ISSUE THREAD

**Issue by Mat2095** [PlitherProject/Plither#28]

**Comment by volconst** [comment]
yes, it packages together existing client and server, so you can run it locally without internet.
The server does not populate with food - not really playable.
The good thing is that the client code snapshot seems compatible with server.
